docs/quick-tutorials/k8s-kafka-mtls.mdx

```diff
@@ -406,37 +409,76 @@ ALL from hosts: *, User:CN=client.otterize-tutorial-kafka-mtls,O=SPIRE,C=US has
 for operations: ALL from hosts: *), zkVersion : 6 (kafka.security.authorizer.AclAuthorizer)
 ```
 
+If you've attached Otterize OSS to Otterize Cloud, go back to see the [access graph in your browser](https://app.otterize.com). Click on the Kafka service, and click at the bottom of it to focus on it and show its details:
+
+![Access graph](/img/quick-tutorials/k8s-kafka-mtls/protected.png)
+
+We can see what happened:
+1. Kafka topic-specific intents from **[client]** are declared (solid green line).
+2. Calls from **[client-other]** are not declared (orange line).
+3. Looking at the Kafka service, we can see that **[client]** has specific access configured (via Kafka ACLs) to perform `all` operations on the `mytopic` topic.
+
+Since discovered intents from the network mapper don't specify what specific topics and operations clients are performing (or attempting to perform), the access graph cannot show information on what is being blocked vs allowed (red vs green). That feature is in development.
+
+Also, the access graph shows information about the mTLS certificates (credentials) distributed to the various services, as long as [Cloud-managed credentials](/security#cryptographic-credentials) are being used. Visibility for certificates distributed through an in-cluster SPIRE is in development.
+
+## Turn on protection
+
+At this point, we haven't actually protected our Kafka broker. From everything we've done so far, we can see, however, that if we were to turn on protection, the `client-other` and `client-authenticated` would lose access to the broker.
+
+Let's see that in action. This only requires a single change to our helm chart's `values.yaml` file to set `allowEveryoneIfNoAclFound: false` which will set a `DEFAULT: deny` ACL. Our clients that have not declared intents will be blocked from accessing the broker.
```

Comment:
This is a Kafka setting, it does not affect which ACLs are set by the intents-operator. If you turn this off, all access to other topics will be blocked.
I think that you can keep this on and the result will be that the tutorial works as described, since there will be an ACL on only these topics, meaning all other access is blocked.

Reply:
To protect our Kafka broker, we should choose: either we use global shadow mode, or we flip `intentsRequired` to `true`, where the broker starts out with enforcement on.

Reply:
@davidgs We changed this to work with allowEveryoneIfNoAclFound, so this change is no longer made.